compiler/plugins/input/Torch/InputConversion/FuncConversion.cpp

```diff
@@ -609,33 +688,38 @@ class FuncConversionPass final
   }
 
   LogicalResult convertFuncOp(func::FuncOp torchFunc,
-                              ConvertedAsyncFunctionInfo &convertedFuncInfo) {
+                              ConvertedFunctionInfo &convertedFuncInfo) {
     IRRewriter rewriter(torchFunc.getContext());
     rewriter.setInsertionPoint(torchFunc);
     Location loc = torchFunc.getLoc();
-    // Determine whether to build pure async or async + sync wrapper.
-    bool generateSyncWrapper = true;
```

Comment:
This is the state in upstream right? The code looks like this should have been optional from the get go